drivers/console/bm_serial.c

```diff
@@ -50,6 +56,14 @@ K_MSGQ_DEFINE(rx_queue, sizeof(bm_msg_t), CONFIG_BM_NUM_FRAMES, 4);
 /* Semaphore for ISR and RX_Task to signal availability of Decoded Frame */
 K_SEM_DEFINE(decode_sem, 0, 1);
 
+/* Firmware Update variables */
+uint32_t _img_length = 0;
```

Comment:
For globals that are only used in a single source file, make them static to limit their linkage visibility.